airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -383,6 +384,59 @@ def path(self, **kwargs) -> str:
         return f"{self.data_field}.json"
 
 
+class ProductsGraphQl(IncrementalShopifyStream):
+    filter_field = "updated_at"
+    cursor_field = "updatedAt"
+    data_field = "graphql"
+    http_method = "POST"
+
+    def path(self, **kwargs) -> str:
+        return f"{self.data_field}.json"
+
+    def request_params(
+        self, stream_state: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None, **kwargs
```

Comment:
long line

Reply:
Formatted via `black -l 120 . --extend-exclude virtualenv` had no effect here. Happy to make it shorter but seems short enough?